src/msw/window.cpp

```diff
@@ -4820,8 +4820,7 @@ namespace wxMSWImpl
 {
 
 AutoSystemDpiAware::SetThreadDpiAwarenessContext_t
-AutoSystemDpiAware::ms_pfnSetThreadDpiAwarenessContext =
-    (AutoSystemDpiAware::SetThreadDpiAwarenessContext_t)-1;
+    AutoSystemDpiAware::ms_pfnSetThreadDpiAwarenessContext = nullptr;
```

Comment:
Changing this means we're going to retry loading it every time instead of just trying it once in case it's not available. Not sure if this actually risks happening, but if it does, this doesn't look like a good change.